.github/workflows/lambda_release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
```

Comment:
I don't think we can have two workflows named "Release". Maybe "Create Release Tags" is better?
```suggestion
name: Lambda Release
```

Reply:
a billion percent. great catch!